tests/scripts/components-configuration-crypto.sh

```diff
@@ -827,16 +827,12 @@ component_test_psa_crypto_config_accel_ecdsa () {
     # Configure
     # ---------
 
-    # Start from default config (no USE_PSA) + TLS 1.3
-    helper_libtestdriver1_adjust_config "default"
+    # Start with full config for maximum coverage (also enables USE_PSA)
+    helper_libtestdriver1_adjust_config "full"
```

Comment:
Note: testing this with `!USE_PSA` was intentional as it's a technically valid config*. We can revisit that decision if we think testing technically-valid-but-not-very-sensible configs is not worth our CI time, but we need to have a discussion first.

*And we have components that test with `USE_PSA` and all three of ECDSA, ECDH, ECJPAKE accelerated.